src/OpenSSL/SSL.py

```diff
@@ -250,6 +251,128 @@
 SSL_CB_HANDSHAKE_START = _lib.SSL_CB_HANDSHAKE_START
 SSL_CB_HANDSHAKE_DONE = _lib.SSL_CB_HANDSHAKE_DONE
 
+
+class X509VerificationCodes:
+    """
+    Success and error codes for X509 verification, as returned by the
+    underlying ``X509_STORE_CTX_get_error()`` function and passed by pyOpenSSL
+    to verification callback functions.
+
+    See `OpenSSL Verification Errors
+    <https://www.openssl.org/docs/manmaster/man3/X509_verify_cert_error_string.html#ERROR-CODES>`_
+    for details.
+    """
+
+    OK = _lib.X509_V_OK
+
+    # cryptography v40.0.0 and .1 releases are missing the X509_V_ERR_* codes;
+    # previously they were exposed but not formally part of the public API.
+    # Once pyOpenSSL has minimal required cryptography version of 41 this code
+    # can be run unconditionally. See
+    # https://github.com/pyca/pyopenssl/issues/1206
+    if hasattr(_lib, "X509_V_ERR_CERT_SIGNATURE_FAILURE"):
```

Comment:
Given that .0 and .1 are excluded in the `install_requires`, do we need this check? Is it reachable?

Reply:
Hmm. 

1. Someone could force override it.
2. Conceivably (though not AFAIK) some other past version might have it.
3. A future version could drop it (but then silently failing is bad).

So either this can be dropped, yeah, or the `setup.py` restriction can be dropped... Having inconsistent APIs based on what's installed is weird, so may as well drop this.